src/github.com/getlantern/flashlight/client/pluggable_transports.go

```diff
@@ -0,0 +1,86 @@
+package client
+
+import (
+	"crypto/tls"
+	"fmt"
+	"net"
+
+	"github.com/Yawning/obfs4/transports/obfs4"
+	"github.com/getlantern/keyman"
+	"github.com/getlantern/tlsdialer"
+
+	"git.torproject.org/pluggable-transports/goptlib.git"
+)
+
+type dialFN func() (net.Conn, error)
+type dialFactory func(*ChainedServerInfo, string) (dialFN, error)
+
+var pluggableTransports = map[string]dialFactory{
+	"":      defaultDialFactory,
+	"obfs4": obfs4DialFactory,
+}
+
```

Comment:
I feel like defaultDialFactory could just use a clarifying comment specifying, for example, that it's just a normal HTTP chained proxy or something along those lines @oxtoacart.